src/thor/timedep_reverse.cc

```diff
@@ -37,6 +37,11 @@ void TimeDepReverse::Clear() {
   }
   edgelabels_rev_.clear();
   adjacencylist_rev_.clear();
+
+  // Set the ferry flag to false
+  has_ferry_ = false;
```

Comment:
I assume this isn't related to the not_thru issue? the `has_ferry` flag was simply missing from this implementation?

Reply:
@mandeepsandhu correct.

Reply:
cc @danpat note the missing has_ferry